sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -9,6 +9,11 @@
 import * as coreClient from "@azure/core-client";
 import * as coreHttpCompat from "@azure/core-http-compat";
 
+export type VectorQueryUnion =
+  | VectorQuery
```

Comment:
Just so I understand for the future, is the base class always included in the union even if usage of the base class may be invalid?

Reply:
This is consistent with how other union types are generated. However, we typically re-export these types with the base type omitted from the function signature.